_posts/2024-05-31-friends-dont-let-friends-package-alone-pyconus-24.md

```diff
@@ -0,0 +1,282 @@
+
+Together we created a guidebook that acknowledges a suite of fantastic tools across the ecosystem while
+still leading users down a streamlined path.
+
+<figure>
+  <img src="{{ site.baseurl }}/images/blog/2024/june/pyopensci-packaging-guide-overview.png" alt="Image with the pyOpenSci flower logo in the upper right hand corner. The image shows the packaging lifecycle. The graphic shows a high level overview of the elements of a Python package. the inside circle has 5 items - user documentation, code/api, test suite, contributor documentation, project metadata / license / readme. In the middle of the circle is says maintainers and has a small icon with people. on the outside circle there is an arrow and it says infrastructure." style="width: 70%;" />
+  <figcaption><i class="fa-solid fa-heart-pulse"></i> The generalized scientific Python package development process. <i class="fa-solid fa-heart-pulse"></i>.</figcaption>
+</figure>
+
+
+## Getting accepted to talk about packaging...  <i class="fa-solid fa-ghost"></i>
+
+I was thrilled and terrified to be accepted to give a talk on the process of creating the pyOpenSci Python packaging guidebook at pyCon US in Pittsburgh, Pennsylvania. My talk was called:
+
+“Friends don’t let friends package alone”
+
+It was all about the process of navigating the Python packaging ecosystem and
+then creating the guide. While you might assume such a talk about be technical,
+in reality, it was all about people.
+
+### Me and the child-like imposter inside
+
+Giving this talk was a big deal for me. Imposter syndrome—the feeling that I
+didn't belong, that I wasn't technical enough to build pyOpenSci — has always been
+my reality. As a trained Landscape Architect, I spent most of my college and early graduate years
+practicing perfect lettering, using markers, fancy pens and colored pencils  to bring creative
+landscape visions to life.
+
+Now, as a Pythonista, coder, data scientist, and tech
+geek, I’m very aware of how much more others know than I do about lots of
+different topics—including packaging.
+
+I wasn’t sure what it would be like to be on that big pyCon US presentation stage, in that big room full of chairs. Would they be empty? Luckily, I was able to focus on my friends sitting in the front rows
+
+Carol, Hugo, Jeremiah, and Inessa - they were all there.
+
+Carol told me to lean into the teacher inside of me. I know that teacher best. My body calmed as I remembered why I was there. To help the community move forward.
+
+And similar to how we created our packaging guide - my friends got me through it.
+
+## The core messages in my talk
+
+In my talk, I shared my experiences as a maintainer of [Stravalib](https://github.com/stravalib/stravalib). How
+challenging it was to figure out the best packaging infrastructure for our team.
+However, that experience of rebuilding stravalib also empowered me to better understand the pain points
```

Comment:
stravalib appears as both 'stravalib' and 'Stravalib' throughout the piece

Reply:
stravalib has no brand but i think we use it lower case throughout except when it begins a sentence so i'll fix locally. thanks!

Reply:
fixed.